lib/ai/assistant.ts

```diff
@@ -1,13 +1,250 @@
-import OpenAI from "openai";
-const openai = new OpenAI();
+const { OpenAI } = require("openai");
+require('dotenv').config();
+
+const openai = new OpenAI; // API key is automatically read from OPENAI_API_KEY env variable
 
 async function main() {
   const assistant = await openai.beta.assistants.create({
-    name: "Task Management Assistant",
-    instructions: "You are an assistant for a task management web application.",
-    tools: [{ type: "code_interpreter" }],
     model: "gpt-3.5-turbo",
+    name: "Lookout Assistant",
+    instructions: `
+      You are an AI-powered chat assistant designed to manage tasks within a web application. Here are guidelines on how to use the available functions:
+
```

Comment:
I don't think we want to add explicit guidelines for when to use which functions. This might be somewhat of a "hardcoding" at this point, since the AI should be able to make those decisions.

In the future, when we implement more complicated prompts like "Which tasks are taking the longest?" or something, we can do something like this.

Reply:
Thanks for this idea! I was researching other's implementations of Assistants and most had given context for what the functions do. However, it does seem that the assistant will have access to the functions even if it doesn't need to use it, and based on the `description` for that function they are able to infer. I went ahead and tested it without, and it seems to be working just fine.